docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -1510,6 +1510,33 @@ Parse data with JSONparse.
 {{/JSONparse}}
 <!-- output: bar -->
 ```
+### {{JSONparseSafe}}
+
+Parse data safely with JSONparseSafe. This helper is similar to the JSONparse helper, but allows handling of incorrect JSON data. Returns `parsed data` when JSON is correct; otherwise, renders an `else` block.
+
+
+#### Parameters
+
+- `json` {String} 

```

Comment:
* call this something else because the string might not be serialized JSON.
* add a description

Reply:
The helper try to parse a string and expected that it contains some serialised data. So, technically, passed data has a string type.